Changes

```diff
@@ -1,3 +1,167 @@
+0.42.0.0
+========
+
+This major release brings significant version upgrades to all supported
+renderers : Arnold 5, 3Delight 13 (NSI), and Appleseed 1.8. It also
+introduces native support for OpenVDB and of course includes the usual
```

Comment:
Does it bring USD too or was that in 0.41 already?

Reply:
At the moment you have make sure Cortex is built with USD support and then manually `import IECoreUSD` to get the SceneInterface registered, only then will the SceneReader know how to deal with USD files. I have a [PR](https://github.com/GafferHQ/dependencies/pull/66) that introduces USD to gafferDependencies, which is just waiting on a fix which is in Don's latest Cortex PR. Once that is merged, I'll import IECoreUSD in a config file in Gaffer, and update the Changes file to reflect the glorious new addition.